add-outdoor_temp-col.ts

```diff
@@ -0,0 +1,27 @@
+import { Pool } from "pg";
```

Comment:
Can you rename this file to `add-outdoor-temp-col.ts` for consistency?

Reply:
Done!